.circleci/config.yml

```diff
@@ -38,24 +37,14 @@ jobs:
         # Note this works even though the submodule hasn't been checkout out yet.
       - run:
           name: Get postgres cache key
-          command: |
-            git rev-parse HEAD:vendor/postgres > /tmp/cache-key-postgres
+          command: git rev-parse HEAD:vendor/postgres > /tmp/cache-key-postgres
```

Comment:
Do you believe this (and similar) change is for the better? Personally, I find this a bit harder to read (those pipes "|" make the next line of code stand out). Also, this causes `git diff` to be slightly bigger than necessary if somebody appends a new line.

Reply:
it's just about a habit. Usually `|` used for multi-line scripts but when you need execute just one command I see no any advantages this notation.